experimental/origin-eventsource/src/index.js

```diff
@@ -84,7 +93,12 @@ class OriginEventSource {
       data.categoryStr = startCase(data.category.replace(/^schema\./, ''))
     }
 
-    data.unitsTotal = data.unitsTotal ? data.unitsTotal - soldUnits : 0
+    units.remaining = data.unitsTotal - (units.sold + units.pending)
```

Comment:
This is technically unnecessary (I could make the calculation in line 98), but I preferred it in case we need to refer to this variable again in the future.